src/helpers/getDistance.ts

```diff
@@ -0,0 +1,8 @@
+import { formatDistance } from "date-fns";
+
+export function getDistance(datePast: string) {
```

Comment:
can we name this getTimeDistance or getTimeAgo or something? That would be a bit more clear